src/test/java/teammates/test/cases/ui/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -423,11 +423,13 @@ public void testEditQuestionAction() throws Exception {
     public void testDeleteQuestionAction() {
         ______TS("rank: qn delete");
```

Comment:
@chowyb Actually I think that may actually be somewhat redundant as the behavior is already tested. Though it must be admitted checking for cancelling is in line with the current testing style (which can be quite brittle).